routes/delegate/user.ts

```diff
@@ -0,0 +1,47 @@
+import { Request, Response } from 'express';
+import { deriveWallet } from './register';
+import { addPaymentDelegationPayee } from '../../lit';
+
+export async function addPayeeHandler(req: Request, res: Response) {
+    const payeeAddresses = req.body as string[];
+    const apiKey = req.header('api-key');
+    const payerSecret = req.header('payer-secret-key');
+
+    if (!apiKey || !payerSecret) {
+        res.status(400).send('Missing or invalid API / Payer key');
+        return;
+    }
+
+    if (!payeeAddresses || !Array.isArray(payeeAddresses) || payeeAddresses.length < 1) {
+        res.status(400).send('Missing or invalid payee addresses');
+        return;
+    }
+
+    const wallet = await deriveWallet(apiKey, payerSecret);
+    let error: string | boolean = false;
+
+    try {
+        const tx = await addPaymentDelegationPayee({
+            wallet,
+            payeeAddresses
+        });
+
+        if (!tx) {
```

Comment:
As in other places, `addPaymentDelegationPayee()` doesn't return `undefined` so this if() check won't fire.  Also, it's not a `tx` :)